package/gluon-respondd/src/respondd.c

```diff
@@ -211,11 +212,21 @@ static struct json_object * get_rootfs_usage(void) {
 	return jso;
 }
 
+static struct json_object * get_time(void) {
+	struct timespec now;
+	clock_gettime(CLOCK_REALTIME, &now);
+
+	struct json_object *jso = json_object_new_int(now.tv_sec);
+
+	return jso;
+}
+
 static struct json_object * respondd_provider_statistics(void) {
 	struct json_object *ret = json_object_new_object();
 
 	json_object_object_add(ret, "node_id", gluonutil_wrap_and_free_string(gluonutil_get_node_id()));
 
+	json_object_object_add(ret, "current_time", get_time());
```

Comment:
just `time`, we can only hope it's always current, and not an outdated one, can we?

Reply:
done